src/ILogger/Readme.md

```diff
@@ -0,0 +1,195 @@
+        {
+            Configuration = configuration;
+            _logger = logger;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);                         
+             _logger.LogInformation("From ConfigureServices. Services.AddMVC invoked"); // This will be picked up up by AI
+        }
+
+        // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
+        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                _logger.LogInformation("Configuring for Development environment");
+                app.UseDeveloperExceptionPage();
+            }
+            else
+            {
+                _logger.LogInformation("Configuring for Production environment");
+            }
+
+            app.UseMvc();
+        }
+    }
+```
+
+```
+public class ValuesController : ControllerBase
+    {
+        private readonly ILogger _logger;
+
+        public ValuesController(ILogger<ValuesController> logger)
+        {
+            _logger = logger;
+        }
+
+        // GET api/values
+        [HttpGet]
+        public ActionResult<IEnumerable<string>> Get()
+        {
+            _logger.LogInformation("This is an information trace..");
+            _logger.LogWarning("This is a warning trace..");
+            _logger.LogTrace("this is a Trace level message");
+            return new string[] { "value1", "value2" };
+        }
+		...
+		....
+	}
+```
+
+In both the above examples, the standalone package Microsoft.Extensions.Logging.ApplicationInsights was used. This, by default, would be using a bare minimum `TelemetryConfiguration` for sending data to
+application insights. Bare minimum means the channel used will be `InMemoryChannel`, no sampling, and no standard TelemetryInitializers. This behavior can be overridden for a console application
+as shown in below example.
+
+Install additional package
+```
+<PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.9.0-beta3" />
+```
+
+Following is the relevant section showing how to override the default `TelemetryConfiguration`. This example configures `ServerTelemetryChannel`, sampling, 
+and a custom TelemetryInitializer.
+
+```
+			// Create DI container.
+            IServiceCollection services = new ServiceCollection();
+            var serverChannel = new ServerTelemetryChannel();
+            services.Configure<TelemetryConfiguration>(
+                        (config) =>
+                        {
+                            config.InstrumentationKey = "ikey";
```

Comment:
This might confuse people as we are specifying IKey twice (earlier in setting the logger up).